.eslintrc.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
   extends: ['@kwizapp/eslint-config-ts-react'],
+  rules: {
+    '@typescript-eslint/member-delimiter-style': 'off',
```

Comment:
Remove this, I had added it temporarily.

```suggestion

```

Reply:
I've added this change in a separate Pull Request.
That one is merged now, so I can adjust here accordingly.